### ISSUE TITLE
Allow to change the email in oauth2 registration

### ISSUE THREAD

**Issue by lasote** [biicode/biicode#17]
Just associate the oauth2 token to account and send the confirmation email if email account is not the same from the oauth2 provider.

**Comment by MariadeAnton** [comment]
Hi @lasote, this may also be related to give users the ability to link different e-mail accounts in biicode, maybe with more than just one-email: https://github.com/biicode/biicode/issues/4

**Comment by lasote** [comment]
Yes, but having multiple emails is much harder. All have to be confirmed, (or in process, so have to keep a state), change the login system to use multiple emails, oauth process...  That's a different feature.

**Comment by lasote** [comment]
Its funny...users want to have multiple accounts with one email and others want single account with multiple emails? LoL  So... by now... One Email to Rule Them All!! XDDD

**Comment by MariadeAnton** [comment]
but what if they don't want to change their biicode base email?
